database/postgresql/generateReviews.js

```diff
@@ -0,0 +1,81 @@
+const fs = require('fs');
+const csvWriter = require('csv-write-stream');
+const faker = require('faker');
+
+const numberOfDishes = 10000064;
```

Comment:
such a specific number!

Reply:
Haha, that's the number of dishes my last script generated.